src/internal/observable/iif.ts

```diff
@@ -90,7 +90,10 @@ import { SubscribableOrPromise } from '../types';
  * @static true
  * @name iif
  * @owner Observable
- */
+*/
+/* tslint:disable:max-line-length */
+export function iif<T = never, F = never>(condition: () => boolean, trueResult?: SubscribableOrPromise<T>, falseResult?: SubscribableOrPromise<F>): Observable<T | F>;
```

Comment:
Is this OK?

Reply:
Honestly, we should probably only make these changes coupled with the dtslint unit tests that clearly show the problem it's solving, and that the tests pass in CI.